component/portal/src/main/java/org/exoplatform/portal/mop/storage/SiteStorageImpl.java

```diff
@@ -316,7 +357,11 @@ private PortalData buildPortalData(SiteEntity entity) {
                           entity.getSkin(),
                           rootContainer,
                           entity.isDefaultSiteBody(),
-                          redirects);
+                          redirects,
+                          entity.isDisplayed(),
+                          entity.getDisplayOrder(),
+                          null,
```

Comment:
bannerUploadId is always null in the retrieving case so it won't be cashed
@azayati  @boubaker

Reply:
Yep, thus to delete as attribute